scripts/deploy.sh

```diff
@@ -214,12 +214,13 @@ if [[ "${DEPLOY_OA}" == "yes" ]]; then
     run_ansible haproxy-install.yml
   fi
 
-  # We have to skip V-38462 when using an unauthenticated mirror
-  # V-38660 is skipped for compatibility with Ubuntu Xenial
+  # NOTE(mhayden): V-38642 must be skipped when using an apt repository with
+  # unsigned/untrusted packages. V-38660 checks for SNMP configs with SNMPv1/2
+  # enabled and it must be skipped all of the time.
```

Comment:
@hughsaunders What should be removed? The only reason we skip it here is that we have SNMPv1/2 configs on RPC hosts. We shouldn't do that, but that's a separate problem to fix. ;)